tests/rbac/application/common/test_jwt_token_provider_impl.py

```
@@ -0,0 +1,17 @@
+    def test_토큰생성검사(self):
+        # Arrange
+        settings = Settings()
+        jwt_token_provider = JwtTokenProviderImpl(settings=settings)
+        payload = {
+            "user": "hello"
+        }
+        # Act
+        token: str = jwt_token_provider.generate_token(payload=payload, ttl=60)
+        # Assert
+        result: bool = jwt_token_provider.validate_token(token=token)
+        assert result == True
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if result:` for truth checks

Replace with `result`

(E712)